### ISSUE TITLE
Bridging in 100% dense areas.

### ISSUE THREAD

**Issue by BagelOrb** [Ultimaker/CuraEngine#1409]
**Application Version**
4.8, latest master, likely any

**Platform**
Windows or WSL

**Steps to Reproduce**
Load 
[UMS5_shear_test_v0.1_p1.3mf.renamed.zip](https://github.com/Ultimaker/CuraEngine/files/5958945/UMS5_shear_test_v0.1_p1.3mf.renamed.zip)

**Actual Results**
Bridging causes wrong skin angles:
![image](https://user-images.githubusercontent.com/8895761/107527279-f85e0d80-6bb8-11eb-913f-0a0cbcc352ab.png)

**Expected results**
Only 45*, -45* alternating skin line directions
![image](https://user-images.githubusercontent.com/8895761/107527526-2d6a6000-6bb9-11eb-8963-be88a0729da2.png)
![image](https://user-images.githubusercontent.com/8895761/107527678-55f25a00-6bb9-11eb-80f3-666fd2afbc2c.png)

**Workaround**
Turn off bridging altogether by enabling Bridge settings and setting Minimal Bridge Wall Length to 9999.

**Additional Information**
Bridging is a monster dinosaur implemented before my time.

**Comment by fvrmr** [comment]
Hi @BagelOrb,
I tried it myself and isn't the solution to just enable the bridge settings and have the `bridge skin support threshold` on 50% which is the default.

![Screenshot (50)](https://user-images.githubusercontent.com/77000924/107644027-30c42100-6c77-11eb-9d2a-61b50905761d.png)

![Screenshot (51)](https://user-images.githubusercontent.com/77000924/107644020-2f92f400-6c77-11eb-8a8d-25dfc1bf72df.png)

Please let me know if this solves your problem.

**Comment by BagelOrb** [comment]
Yeah it seems the whole problem is circumvented when I just enable bridge settings.

Why is it even an option to leave the bridge settings disabled?

**Comment by Ghostkeeper** [comment]
Because the newer bridging routine is not as well tested for print quality as the old one. It's not just more settings but a different algorithm to detect bridges and orient the lines, too. We have a ticket in our backlog (CURA-6042) to remove the old one in favour of the one currently in experimental. In my experience the new bridging routine works better in most cases.

A related problem is that the implementation of the new bridging routine is in a bit of a weird place. It wouldn't be a big refactor but it's a maintenance issue. Of course, the maintenance issue is even greater now that we need to maintain two implementations... So yeah it's on our backlog. I'd like to have the bridging settings always enabled and remove that setting and the old implementation.

**Comment by BagelOrb** [comment]
So CURA-6042 would solve this bug. Would be great to have it implemented then!
